aws/resource_aws_db_instance_test.go

```diff
@@ -1932,6 +1924,17 @@ func testAccCheckAWSDBInstanceExists(n string, v *rds.DBInstance) resource.TestC
 	}
 }
 
+func testAccAWSDBInstanceImportStateIdFuncARN(resourceName string) resource.ImportStateIdFunc {
```

Comment:
I don't see this method used anywhere?

Reply:
Heh whoops -- that was related to what I was looking into. I'll remove it! 👍